src/classes/Foo.cls

```diff
@@ -0,0 +1,16 @@
+public class Foo {
+    void foo() {
+        
+        String myString = 'MY_HARDCODED_STRING';
+        String myString2 = 'MY_HARDCODED_STRING';
+        String myString3 = 'MY_HARDCODED_STRING';
+        
+        Blob hardCodedIV = Blob.valueOf('Hardcoded IV 123');
+        Blob myKey = Blob.valueOf(myString);
+        Blob data = Blob.valueOf('Data to be encrypted');
+        Blob encrypted = Crypto.encrypt('AES128', myKey, hardCodedIV, data);
```

Comment:
This encryption key is not randomized. Please ensure all encryption keys are randomized, for example using methods such as `Crypto.generateAesKey`.